### ISSUE TITLE
Node.js => Error: connect ECONNREFUSED

### ISSUE THREAD

**Issue by n-lavrenko** [IndicoDataSolutions/IndicoIo-node#118]
Hi!
I did'n know what the problem it may be cause it working on my machine yesterday. 
It's simple code from tutorial: 

```
var indico = require('indico.io');
indico.apiKey =  'f92bd5647cc7536bf75518a3eebce323';

var response = function(res) { console.log(res); }
var logError = function(err) { console.log(err); }

// single example
indico.sentiment({data: 'I love writing code!'})
  .then(response)
  .catch(logError);
```

on response i receive:

```
{ [Error: connect ECONNREFUSED 104.197.152.141:80]
  cause:
   { [Error: connect ECONNREFUSED 104.197.152.141:80]
     code: 'ECONNREFUSED',
     errno: 'ECONNREFUSED',
     syscall: 'connect',
     address: '104.197.152.141',
     port: 80 },
  isOperational: true,
  code: 'ECONNREFUSED',
  errno: 'ECONNREFUSED',
  syscall: 'connect',
  address: '104.197.152.141',
  port: 80 }
```

I was tried to reinstall Node.js, tried version 6.3.1 and 4.4.7 of Node.js. Tried to send api call from PostMan - it works. But from node.js code - not working. Tried to debug - but nothing found yet. 

<img width="1440" alt="screen shot 2016-08-14 at 05 02 18" src="https://cloud.githubusercontent.com/assets/7901225/17646660/5bab8566-61dc-11e6-96fc-e7b13f91ca8f.png">

<img width="1440" alt="screen shot 2016-08-14 at 05 20 55" src="https://cloud.githubusercontent.com/assets/7901225/17646730/fbd0ebce-61de-11e6-85e7-a514ec48b61f.png">

Please, help me. 
Thanks.

**Comment by sihrc** [comment]
What version of this library are you currently using?

**Comment by n-lavrenko** [comment]
Latest version:  0.10.0

**Comment by sihrc** [comment]
Thanks @n-lavrenko for creating this issue. We've just deployed a fix in version 0.10.1.
Sorry about that!

**Comment by n-lavrenko** [comment]
Thanks @sihrc!
It's ok, each program contain the bug ;)
Finally I can start developing!
